modules/nf-core/gatk4/collectreadcounts/main.nf

```diff
@@ -30,14 +30,14 @@ process GATK4_COLLECTREADCOUNTS {
                     args.contains("--format TSV")  ? "tsv" :
                     "hdf5"
 
-    def avail_mem = 3
+    def avail_mem = 3072
```

Comment:
do we actually need to note down that it is in mb somewhere? Since all labels etc use GB

Reply:
Yeah, nextflow handles all of the conversions